cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -714,7 +715,81 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
                 AbstractVisibilityEditor.prototype.getContext.call(this),
                 {
                     hide_after_due: this.modelVisibility() === 'hide_after_due',
-                    self_paced: this.model.get('self_paced') === true
+                    self_paced: course.get('self_paced') === true
+                }
+            );
+        }
+    });
+
+    ShowCorrectnessEditor = AbstractEditor.extend({
+        templateName: 'show-correctness-editor',
+        className: 'edit-show-correctness',
+        events: {
+            'change input[name=show-correctness]': 'togglePastDueWarning'
+        },
+
+        modelValue: function() {
+            var showCorrectness = this.model.get('show_correctness');
+            if (showCorrectness === 'always' ||
```

Comment:
How could showCorrectness be anything other than 'always', 'never', or 'past_due'?